python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -36,6 +36,8 @@
 from pyspark.testing.utils import ReusedPySparkTestCase
 from pyspark.testing.connectutils import should_test_connect, connect_requirement_message
 from pyspark.testing.pandasutils import PandasOnSparkTestCase
+from pyspark.sql.connect.client import Retrying
```

Comment:
(all these import should be under `if should_test_connect` at L48 below)

Reply:
Fixed.